Products/EEAContentTypes/browser/language.py

```diff
@@ -91,6 +91,19 @@ class LanguageSelectorData(BrowserView):
     """ VIEWified languageSelectorData.py from LinguaPlone/skins/
         so we can test it
     """
+    
+    def has_translations(self, context):
+        """ check if context has published translations
+        """
+        published = {}
```

Comment:
This code was present until now as we can see from the removed code diff on the left.
I am confused thought of why this logic works since on non anon you get all of the translations
of the context while anonymous **translations** is assigned only 1 value that is published.